### PR TITLE
gui: Add toolTip and placeholderText to sign message fields

### PR DESCRIPTION
**Pull request by dannmat**
When using the Verify Message functionality, I found the input boxes to be rather confusing as they had no guidance for their purpose.

I have added tooltips and labels to aid users when verifying messages in future

### CODE REVIEW

File: src/qt/signverifymessagedialog.cpp [hunk 1]
```diff
@@ -41,6 +41,9 @@ SignVerifyMessageDialog::SignVerifyMessageDialog(const PlatformStyle *_platformS
     GUIUtil::setupAddressWidget(ui->addressIn_SM, this);
     GUIUtil::setupAddressWidget(ui->addressIn_VM, this);
 
+    ui->messageIn_VM->setPlaceholderText(tr("Enter the signed message to verify"));
```
Comment (promag) [inline src/qt/signverifymessagedialog.cpp]: Why aren't these in the .ui file?

Comment (dannmat) [reply src/qt/signverifymessagedialog.cpp]: I tried to keep the changes consistent with the rest of this file, by adding the placeholder text in code

Comment (laanwj) [reply src/qt/signverifymessagedialog.cpp]: FWIW the historical reason for this is that there was some Qt 4.x version that didn't support placeholder text in the `.ui` file. Support for Qt4 was dropped ages ago so this is not a concern anymore. Feel free to simply add it in the ui file.

Comment (dannmat) [reply src/qt/signverifymessagedialog.cpp]: Ahh that makes sense, thank you. I'll redo these changes tonight in the UI file and commit.

**Comment by dannmat** [comment]
I have changed the placeholder text to be in the UI file instead as per @promag & @laanwj request

**Comment by MarcoFalke** [comment]
Usually, we squash the commits after a fixup, according to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by laanwj** [comment]
ACK after squash

**Comment by dannmat** [comment]
Rebase and Squash done.

**Comment by promag** [comment]
Code review ACK 610d9384de7f4de861d94c9b1af4fddc8aa57ad9.

**Comment by MarcoFalke** [comment]
ACK 610d9384de7f4de861d94c9b1af4fddc8aa57ad9 (looks good, didn't compile or tested the changes)

**Review by fanquake — APPROVED** [review]
ACK - 610d9384de7f4de861d94c9b1af4fddc8aa57ad9

master (029c65e04cc2826817e6b5b10696c0e390694ae8):
master


PR (610d9384de7f4de861d94c9b1af4fddc8aa57ad9):
pr

**Pull request merged**